testdata/tree/meta_tree/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "enucleate-testdata-meta-tree"
```

Comment:
One small thing: `enucleate` was the old working title. Please change it to `cargo-mutants-testdata...` here. I just fixed the other tests.